src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkRepositoryContext.java

```diff
@@ -749,8 +749,22 @@ public StructImpl download(
                     + " archive. Instead of needing to specify this prefix over and over in the"
                     + " <code>build_file</code>, this field can be used to strip it from extracted"
                     + " files."),
+        @Param(
+            name = "renameFiles",
```

Comment:
This is the first time I actually noticed that `stripPrefix` has a camel case name - that looks like a bug as Starlark parameters usually use snake case. Using snake case for `renameFiles` at least wouldn't complicate future efforts to clean up this inconsistency.

Reply:
Done.